lib/index.js

```diff
@@ -0,0 +1,261 @@
+'use strict';
+
+const path = require('path');
+const querystring = require('querystring');
+const fetch = require('node-fetch');
+const _ = require('lodash');
+
+const defaults = require('./defaults');
+
+const conditionalTranslate = (key, t) => {
+  let result = t(key);
+  if (result === key) {
+    result = null;
+  }
+  return result;
+};
```

Comment:
If this isn't already an option in `i18n-future` then I *really* need to do that.